charts/cray-import-kiwi-recipe-image/values.yaml

```diff
@@ -1,30 +1,27 @@
-
 # Full image including repo, e.g. dtr.dev.cray.com/cray/import-image:1.2.3
-#                                         ^         ^         ^        ^
-#                                         |         |         |        |
-#                    ['import_image'] [registry]  [repo]    [name]   [tag]
+#                                                   ^                  ^
+#                                                   |                  |
+#                                             ['repository]          [tag]
 
 # Docker image that will be used in the `import_job` (settings below). This image
 # should use the cray-ims-load-artifacts image as its base.
 import_image:
-  registry: "dtr.dev.cray.com"
-  repo: cray
-  name: ""
-  tag: ""
+  image:
+    repository: ""
+    tag: ""
+    pullPolicy: IfNotPresent
 
 # Docker image that will update the product catalog with results of the import
 # of images and recipes. Specifies the product catalog ConfigMap as well.
 catalog:
   image:
-    registry: "dtr.dev.cray.com"
-    repo: cray
-    name: cray-product-catalog-update
+    repository: artifactory.algol60.net/csm-docker/stable/cray-product-catalog-update
     tag: 1.2.50
```

Comment:
There will be a 1.3.x coming in https://github.com/Cray-HPE/cray-product-catalog/pull/17

Reply:
Gottcha. Let me know when we have a version number.

Reply:
1.3.1 is building now.